docs/src/design/story/bundles/credentials/EXTENSIONS.md

```
@@ -0,0 +1,512 @@
+    issue_date: "2026-04-12",
+    expiry: "2027-04-12",
+    purpose: "merchant",
+    photo_url: "..."
+  },
+  hints: { label_text: "Permit (Imperial)" }
+}
+```
+
+The CLI port renders the packet as:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)